Change airDatepickerInput's button display to table-cell

Mkranj opened the pull request:
# Reason for change
When using `airDatepickerInput` in sidebars and enabling the `addon` button, the widget's display breaks down. The button is not attached to the input, and sometimes it doesn't get colored properly. This happens in the sidebar, but not in the main body. Noticed when used in `shinydashboard::dashboardPage()`.

# Bug screenshot
I've attached an example of the behavior before and after changes in this PR.
<img width="899" alt="comparison" src="https://github.com/dreamRs/shinyWidgets/assets/111303584/b707ad12-3e87-4b8c-9ed3-6b7fb2c74221">


# Code Example
Try this app and check the layout in the current build. Then compare it with the version in this PR.
```
library(shiny)
library(shinyWidgets)
library(shinydashboard)

ui <- dashboardPage(
  dashboardHeader(),
  dashboardSidebar(
    airDatepickerInput(inputId = "side_left", addon = "left"),
    airDatepickerInput(inputId = "side_right", addon = "right")
    ),
  dashboardBody(
    airDatepickerInput(inputId = "body_left", addon = "left"),
    airDatepickerInput(inputId = "body_right", addon = "right")
    )
)

server <- function(input, output, session) {
  
}

shinyApp(ui, server)
```

## Alternate solution
Instead of change the div style, a CSS class could be added to the divs containing addon buttons. Then the display property could be changed for that class.
Let me know if this is a more preferable solution than inline CSS styling, and I will make this change.

pvictor commented:
Hello,
Thanks for fixing this!
Yes it will be better to use a CSS class I think, the CSS files with custom styles for airDatepicker is here : https://github.com/dreamRs/shinyWidgets/blob/master/srcjs/css/air-datepicker.css

The Javascript/CSS for airDatepicker  is managed by [{packer}](https://github.com/JohnCoene/packer), so you need to install nodejs if you want to bundle the assets, but if you modify air-datepicker.css I can do the bundle if you want.

Install nodes modules with:

```r
packer::npm_install()
```

Modify `srcjs/css/air-datepicker.css`, then run:

```r
packer::bundle()
```

Re-install R package and try `airDatepickerInput()` function.

Mkranj commented:
Hi, thank you for the comment.
I've changed the code so the buttons have a new CSS class `dp-addon` and modified the .css file to target the buttons.
I don't have NodeJS installed, could you please make the bundle?

pvictor commented:
Perfect 👌 , thank you !
